### ISSUE TITLE
[Feature request] Seafile and ownCloud support

### ISSUE THREAD

**Issue by fossxplorer** [exo-addons/cloud-drive-extension#33]
Would be great to be able to access files from Seafile or ownCloud too :)

**Comment by pnedonosko** [comment]
Yep :) ownCloud already requested in #17. I'll put this in "new features queue". But we are open for cooperation with enthusiasts and/or cloud vendors who interested in support of their document platforms in eXo. Cloud Drive has Connector API, its v1 already released, it can be used to create own eXo add-ons that will plug to the core add-on and provide support for more documents services.
